nvidia_nims/vista_3d_remote_nim.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        invoke_url = "https://health.api.nvidia.com/v1/medicalimaging/nvidia/vista-3d"
</pre><p>I would turn this into an argument for the function, and where you call this function pass in this URL explaining what it is, and maybe suggest other models that would work here too to do something different. I think that helps illustrate what the end point is in the notebook by explicitly using it in a cell rather than being hard coded in a function. At the top of the notebook I would also provide very brief descriptions of what VISTA 3D and NIM are, but very brief next to the URLs you provide for each. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Quite a bit of this function is specific to the VISTA 3D NIM and this web interface/URL (e.g., the validation logic and the URL request parameters/arguments).   So, I don't think the URL should be a parameter to this function.

However, I agree that a brief description of VISTA 3D and NIMs are needed, and I will add them.